windows-nsis/openvpn.nsi

```diff
@@ -228,8 +231,15 @@ Section /o "-workaround" SecAddShortcutsWorkaround
 	; as we don't want to move SecAddShortcuts to top of selection
 SectionEnd
 
-Section /o "${PACKAGE_NAME} User-Space Components" SecOpenVPNUserSpace
+Section /o "-launchondummy" SecLaunchGUIOnLogon0
```

Comment:
This is because nsis does not allow to refer to a section before its defined. We do not want to move these sections up as then they will then get listed at the top. If we hide them we could move them up and get rid of the dummies. Currently I have added them to "Advanced" indicating that a user normally should leave them alone. And we want "Advanced" to appear at the bottom of the options list.

Is there a better way of achieving this?